diff --git a/node_modules/typescript/lib/typescript.d.ts b/node_modules/typescript/lib/typescript.d.ts
index 0c17632..f251e34 100644
--- a/node_modules/typescript/lib/typescript.d.ts
+++ b/node_modules/typescript/lib/typescript.d.ts
@@ -7604,4 +7604,26 @@ declare namespace ts {
     const isIdentifierOrPrivateIdentifier: (node: Node) => node is MemberName;
 }
 
+// for patch-package
+declare namespace ts {
+    export interface Node extends ReadonlyTextRange {
+        readonly type?: TypeNode,
+        readonly symbol?: Symbol
+    }
+
+    export interface TypeNode extends Node {
+        _typeNodeBrand: any;
+        // Maybe this should be in Node type
+        locals?: ESMap<Symbol, string> | undefined;
+    }
+
+    export interface Type {
+        symbol?: Symbol;
+        types?: Type[],
+        resolvedTypeArguments?: Type[],
+        value?: unknown,
+        node?: Node
+    }
+}
+
 export = ts;
\ No newline at end of file
